imap_processing/swe/l1a/swe_science.py

```diff
@@ -133,18 +133,18 @@ def swe_science(decom_data):
         # where 1260 = 15 seconds x 12 energy steps x 7 CEMs
         # Take the "raw_counts" indices/counts mapping from
         # decompression_table and then reshape the return
-        uncompress_data = np.take(decompression_table, raw_counts).reshape(15, 12, 7)
-        raw_counts = raw_counts.reshape(15, 12, 7)
+        uncompress_data = np.take(decompression_table, raw_counts).reshape(180, 7)
+        raw_counts = raw_counts.reshape(180, 7)
```

Comment:
Update the comment above to indicate this is 180 now? Leave the 15 x 12 information though since that still seems useful.